### DISCUSSION TITLE
Lab2-2 FIR Stream axi_dma_out 的命名 (Vivado Block Design)

### DISCUSSION THREAD

**Discussion by BrianEE07** [bol-edu/HLS-SOC-Discussions#66] [Q&A]
如題，在 tb FIRN11Stream.ipynb 中，是呼叫 axi_dma_out_1，但我照 workbook 中的指示命名為 axi_dma_out_0 生成 bitstream，並且在改過 tb 後（1 改成 0）可以順利執行。想請問有需要配合 tb 重新合成後繳交嗎？還是可以交 axi_dma_out_0 的版本，謝謝。
<img width="508" alt="Screenshot 2023-09-23 at 18 20 16" src="https://github.com/bol-edu/HLS-SOC-Discussions/assets/46804360/c9dde7cb-d8df-4eab-9abe-14b29bc78b08">
<img width="640" alt="Screenshot 2023-09-23 at 18 23 28" src="https://github.com/bol-edu/HLS-SOC-Discussions/assets/46804360/44daabdb-a1aa-49ea-b889-678e2a92a91f">

**Comment by BarryJu1999** [comment]
不好意思，想詢問同學不知道您是否有詢問過助教這邊該如何修正並繳交作業嗎？
謝謝同學。

**Comment by BrianEE07** [comment]
我沒特別去問誒。後來直接照測資的 axi_dma_out_1 重新用 vivado 合成一版 XD

**Reply by BarryJu1999** [reply]
了解，謝謝同學。
